content/2.components/hyper-text.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Hyper Text
+description: A hyper changing text animation as you hover..
+---
+
+::ComponentLoader{label="Preview" componentName="HyperTextDemo" type="examples"}
+::
+
+## API
+
+| Prop Name       | Type      | Default  | Description                                       |
+| --------------- | --------- | -------- | ------------------------------------------------- |
+| `class`         | `string`  | `""`     | Additional CSS classes to apply to the component. |
+| `text`          | `string`  | Required | Text to animate                                   |
+| `duration`      | `number`  | `80`    | The duration for which the animation runs.        |
+| `animateOnLoad` | `boolean` | `true`   | Play animation on load                            |
+
+## Component Code
+
+You can copy and paste the following code to create this component:
+
+::CodeViewer{filename="HyperText.vue" language="vue" componentName="HyperText" type="ui" id="hyper-text"}
+::
+
+## Credits
+
+-   Inspired by [Magic UI's Hyper Text](https://magicui.design/docs/components/hyper-text) component.
```

Comment:
Add your name with profile link in credits section.

Reply:
Done.